Vagrantfile

```diff
@@ -0,0 +1,99 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = '2'
+
+@script = <<SCRIPT
+# Install dependencies
+
+add-apt-repository ppa:ondrej/php -y
+apt-get update
+apt-get install -y apache2 php5.6 php5.6-curl php5.6-sqlite php5.6-xdebug curl php5.6-curl zip unzip
+apt remove -y php7.1-cli
+
+# Configure Apache
+echo "<VirtualHost *:8080>
+	DocumentRoot /var/www/web
+	AllowEncodedSlashes On
+
+	<Directory /var/www/web>
+        Options +FollowSymLinks -MultiViews
+        DirectoryIndex index.php index.html
+        Order allow,deny
+        Allow from all
+        AllowOverride All
+        RewriteEngine On
+        RewriteCond %{REQUEST_FILENAME} !-d
+        RewriteCond %{REQUEST_FILENAME} !-f
+        RewriteRule ^ index.php [QSA,L]
+	</Directory>
+
+
+	ErrorLog ${APACHE_LOG_DIR}/error.log
+	CustomLog ${APACHE_LOG_DIR}/access.log combined
+</VirtualHost>
+<VirtualHost *:8081>
+	DocumentRoot /var/www/web
+	AllowEncodedSlashes On
+
+	<Directory /var/www/web>
+        Options +FollowSymLinks -MultiViews
+        DirectoryIndex index.php index.html
+        Order allow,deny
+        Allow from all
+        AllowOverride All
+        RewriteEngine On
+        RewriteCond %{REQUEST_FILENAME} !-d
+        RewriteCond %{REQUEST_FILENAME} !-f
+        RewriteRule ^ index.php [QSA,L]
+	</Directory>
+
+
+	ErrorLog ${APACHE_LOG_DIR}/error.log
+	CustomLog ${APACHE_LOG_DIR}/access.log combined
+</VirtualHost>" > /etc/apache2/sites-available/000-default.conf
+
+a2enmod rewrite
+echo "xdebug.remote_enable=on" >> /etc/php/5.6/apache2/conf.d/xdebug.ini
+echo "xdebug.remote_host=192.168.34.1" >> /etc/php/5.6/apache2/conf.d/xdebug.ini
```

Comment:
This address seems oddly specific (`192.168.34.1`). Why is that?

Reply:
I believe that 33 is the default.  I could be wrong.

Reply:
33 definitely used to be/possibly is still the default.  Doing a quick Google search returns tons of documents with `192.168.33.1`.